app/views/questionnaire.py

```diff
@@ -124,7 +125,12 @@ def get_block(routing_path, schema, metadata, answer_store, eq_id, form_type, co
         return _redirect_to_location(collection_id, eq_id, form_type, next_location)
 
     block = _get_block_json(current_location, schema, answer_store, metadata)
-    context = _get_context(routing_path, block, current_location, schema)
+
+    placeholder_pointers = find_pointers_containing(block, 'placeholders')
+    placeholder_renderer = PlaceholderRenderer(block, answer_store=answer_store, metadata=metadata)
+    replaced_block = placeholder_renderer.render(placeholder_pointers)
```

Comment:
I don't think we should be exposing the pointers functionality here, it seems like something that should be internal to the Renderer. I'd expect to see something like `replaced_block =  PlaceholderRenderer(block, answer_store=answer_store, metadata=metadata).render() ` here.